jekyll/_cci2/oom.md

```diff
@@ -0,0 +1,130 @@
+---
+layout: classic-docs
+title: "Avoiding and Debugging Java Memory Errors"
+description: "How to avoid and debug Java memory errors on CircleCI."
+---
+
+How to avoid and debug Java memory errors on CircleCI.
+
+## Overview
+
+The [Java Virtual Machine](https://en.wikipedia.org/wiki/Java_virtual_machine) (JVM) is a framework
+used for Android and Java development.
```

Comment:
not exactly......

if you're looking for a short, 1-sentence explanation of what the JVM is, how about something like : 

> The Java Virtual Machine (JVM) provides a portable execution environment for various types of Java-based applications.

(adapted from https://javaworld.com/article/3272244/core-java/what-is-the-jvm-introducing-the-java-virtual-machine.html which seems really good)

Reply:
Much more accurate, thanks!